api/models.py

```diff
@@ -39,17 +46,26 @@ class ClientsModel(db.Model):
 
     # Table attributes:
     id = db.Column(db.Integer, primary_key=True)
+    num_listings = db.Column(db.Integer)
+
+    # General Client information:
     client_name = db.Column(db.String(MAX_CREDENTIAL_LEN))
-    client_info = db.Column(db.Text)
+    client_addr = db.Column(db.String(MAX_CREDENTIAL_LEN))
+    client_email = db.Column(db.String(MAX_CREDENTIAL_LEN))
+    client_phone = db.Column(db.String(MAX_CREDENTIAL_LEN))
     num_listings = db.Column(db.Integer)
 
-    def __init__(self, client_info: str, client_name: str, num_listings: int):
+    def __init__(self, client_name: str, client_addr: str, client_email: str,
+                client_phone: str, num_listings: int):
         self.client_name = client_name
-        self.client_info = client_info
+        self.client_addr = client_addr
+        self.client_email = client_email
+        self.client_phone = client_phone
         self.num_listings = num_listings
 
     def __repr__(self):
-        return f'<Client {self.c_name}>'
+        return f'<Client {self.client_name}>'
```

Comment:
Probably want the `id` here too so that the output is unique for different clients.